### PR TITLE
chore(card-browser): more logs

### PR DESCRIPTION
**Pull request by david-allison**
* moved setup higher in setUp to capture more
* More diagnostics for https://github.com/ankidroid/Anki-Android/issues/16541

### CODE REVIEW

File: AnkiDroid/src/main/java/com/ichi2/libanki/ChangeManager.kt [hunk 1]
```diff
@@ -73,13 +77,15 @@ object ChangeManager {
...
         }
+        expired.size.ifNotZero { size -> Timber.v("removing %d expired subscribers", size) }
         for (item in expired) {
             subscribers.remove(item)
         }
     }
 
     @VisibleForTesting
     fun clearSubscribers() {
+        Timber.d("clearing %d subscribers", subscribers.size)
```
Comment (david-allison) [inline AnkiDroid/src/main/java/com/ichi2/libanki/ChangeManager.kt:88]: This explicitly logs even if zero

File: AnkiDroid/src/main/java/com/ichi2/libanki/ChangeManager.kt [hunk 2]
```diff
@@ -73,13 +77,15 @@ object ChangeManager {
                 ref.opExecuted(changes, handler)
             }
         }
+        expired.size.ifNotZero { size -> Timber.v("removing %d expired subscribers", size) }
```
Comment (lukstbit) [inline AnkiDroid/src/main/java/com/ichi2/libanki/ChangeManager.kt:80]: Is the usage of the verbose level intentional?

**Review by lukstbit — APPROVED** [review]
LGTM.
Left a question, feel free to merge otherwise I'm going to merge it when I come back.

File: AnkiDroid/src/main/java/com/ichi2/libanki/ChangeManager.kt (hunk 2 above)
Comment (david-allison) [reply AnkiDroid/src/main/java/com/ichi2/libanki/ChangeManager.kt:80]: Intentional - it happens a lot, and except for debugging this issue, it's low value

**Pull request merged**